docs/fenix.md

```diff
@@ -0,0 +1,58 @@
+# Fenix
+
+It is sometimes useful to test AMO with a Fenix build pointing to -dev or -stage given the tight integration between these two components. In the sequel of this page, we describe how to construct a custom Fenix build.
```

Comment:
sorry this still doesn't make any more sense to me - for there to be a sequel to something there would be an original.  (If so, what is the original and can we link to it.)  And sequels are generally fictional pieces of work anyway.

Reply:
ha I see, ok

Reply:
feel free to make actual suggestions btw, I am not a native speaker and I thought we could use sequel as in "the rest of the content in the page"

Reply:
feel free to make actual suggestions btw, I am not a native speaker and I thought we could use sequel as in "the rest of the content in the page" 😉

Reply:
I think "following" is the word rather then sequel.  You can say "In the following, we..."